core/include/config.h

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright 2017 Intel
```

Comment:
When looking at  https://github.com/zephyrproject-rtos/zephyr/blob/ff07fc7f0a1942610c57a8e041cfbcd171299789/include/misc/util.h#L2
is says:
`Copyright (c) 2011-2014, Wind River Systems, Inc.`

Reply:
I looked at the commit message which introduced the `IS_ENABLED`, not at the file copyright.

Reply:
Fair enough. Please explain that in the commit message. You can drop the "All rights reserved" line too since that's not in that commit any way.

Reply:
Okay.

Reply:
It seems that Intel normally write it as:
`Copyright (c) 2017 Intel Corporation`

Reply:
Indeed, I'll force push the fixup commit.